platform/mellanox/mlnx-platform-api/sonic_platform/fan.py

```diff
@@ -15,23 +15,28 @@
 except ImportError as e:
     raise ImportError (str(e) + "- required module not found")
 
-LED_ON = 1
-LED_OFF = 0
+LED_ON = '1'
+LED_OFF = '0'
 
 PWM_MAX = 255
 
 FAN_PATH = "/var/run/hw-management/thermal/"
 LED_PATH = "/var/run/hw-management/led/"
+# fan_dir only exist Spectrum 2 switches
```

Comment:
now i am confused. 
you mentioned previously fan direction is supported only for spectrum-1 for now and not it is mentioned it only exist in spectrum-2. what am i missing?

Reply:
It's a typo and I just fixed it. FAN's direction isn't supported on spectrum-1.